pkg/common/cluster/clusterutil.go

```diff
@@ -29,9 +29,12 @@ import (
 
 const (
 	// errorWindow is the number of checks made to determine if a cluster has truly failed.
-	errorWindow = 20
+	errorWindow         = 20
+	pendingPodThreshold = 10
```

Comment:
Can you document this constant?

Reply:
I've updated the PR with a description on this.